src/style.css

```
@@ -0,0 +1,113 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+}
+
+.form-big {
+  /*   display: flex;
+  align-items: center;
+  width: 100%; */
+  height: 70px;
+  /*   margin: 20px 0; */
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+}
+
+.form-small {
+  /* display: flex;
+  align-items: center;
+  width: 100%; */
+  height: 42px;
+  /* margin: 20px 0; */
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+}
+
+.input {
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.input::placeholder {
+  color: inherit;
+  font-weight: 300;
+  font-style: normal;
+}
+
+.input-big {
+  font-family: "Avenir", sans-serif;
```

move font-family to `html` tag selector and let inputs inherit it
```suggestion
  font-family: inherit;
```